### PR TITLE
Add organizer to eventitem. Mobil optimization

### PR DESCRIPTION
**Pull request by yepzdk**
https://jira.itkdev.dk/browse/JOBCAMP-90

Show organizers on eventitem

### CODE REVIEW

File: src/components/event/EventItem.jsx [hunk 1]
```diff
@@ -16,30 +16,37 @@ function Event ({ event, onLike }) {
...
       <Card.Body>
         <Row>
-          <Col xs={8}>
+          <Col>
             <Link to={`/event/${event.id}`}>
               <h3>{event.title}</h3>
+              {event.organizers &&
+                <p>
+                  {event.organizers.map(organizer => (
+                    <span key={organizer.id}>{organizer.title}</span>
```
Comment (rimi-itk) [inline src/components/event/EventItem.jsx:25]: Maybe we should add a link to the organizer details page here? 
Do we get (implicit) spacing between multiple organizers or should we add some?

**Review by rimi-itk — COMMENTED** [review]
A couple of questions added.

File: src/components/event/EventItem.jsx (hunk 1 above)
Comment (yepzdk) [reply src/components/event/EventItem.jsx]: You are absolutely right. I've updated the PR

File: src/components/event/EventItem.jsx [hunk 2]
```diff
@@ -16,30 +16,37 @@ function Event ({ event, onLike }) {
...
         <Row>
-          <Col xs={8}>
+          <Col>
             <Link to={`/event/${event.id}`}>
               <h3>{event.title}</h3>
             </Link>
+            {event.organizers &&
+              <p>
+                {event.organizers.map(organizer => (
+                  <span key={organizer.id} className='mr-3'>{organizer.website_url ? <a href={organizer.website_url}>{organizer.title}</a> : organizer.title}</span>
```
Comment (rimi-itk) [inline src/components/event/EventItem.jsx:26]: I was actually thinking about an internal link here (`/organizer/${organizer.id}`) for consistency with stuff we show in the event list.

**Review by rimi-itk — CHANGES_REQUESTED** [review]

**Review by rimi-itk — APPROVED** [review]

**Pull request merged**